platform/ext/target/nordic_nrf/nrf5340/partition/flash_layout.h

```diff
@@ -177,7 +177,7 @@
 /* Number of PS_SECTOR_SIZE per block */
 #define PS_SECTORS_PER_BLOCK   (0x1)
 /* Specifies the smallest flash programmable unit in bytes */
-#define PS_FLASH_PROGRAM_UNIT  (0x1)
+#define PS_FLASH_PROGRAM_UNIT  4
```

Comment:
What about nRF9160? I think we need similar corrections for that platform.

Reply:
Done